client/utils/reduxFormUtils.js

```diff
@@ -15,6 +15,9 @@ export const domOnlyProps = ({
   ...domProps }) => domProps;
 /* eslint-enable */
 
+// eslint-disable-next-line max-len
+const EmailRegex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/i;
```

Comment:
this is a nit-picky personal style thing, but i prefer constants to be all uppercase (i.e. EMAIL_REGEX).

Reply:
@catarak Done changes :)

Reply:
😸 thanks for humoring me

Reply:
@catarak Can it be merged now? 😄